segmentation/requirements.txt

```diff
@@ -1,14 +1 @@
-torch==1.11.0
-torchvision==0.12.0
-click
-numpy
-einops
-python-hostlist
-tqdm
-requests
-pyyaml
-timm == 0.5.4
-mmcv==1.3.8
-mmsegmentation==0.14.1
 tensorboard
```

Comment:
Shouldn't tensorboard be part of the setup.py file too?
Also, within the classification directory there is a requirements.txt with similar packages. 
Maybe we should remove it too for consistency?

Reply:
> Shouldn't tensorboard be part of the setup.py file too?
Yeah, maybe we should do that for simplicity. `tensorboard` isn't a dependency of the `myrtle_vision` library though, only the segmentation scripts. But we do already have classification script specific requirements in the setup.py (like `scikit-learn`).

> Also, within the classification directory there is a requirements.txt with similar packages.
Good catch, yes, let's get rid of that since it's the same packages as the setup.py.